frontend/src/Components/AlgInputs/AlgInputsLoadControl.js

```diff
@@ -1,18 +1,18 @@
-import React, {Component} from "react";
-import { makeStyles } from "@material-ui/core/styles";
-import TextField from "@material-ui/core/TextField";
-import Button from "@material-ui/core/Button";
-import { withStyles } from "@material-ui/core/styles";
-import { countyRes } from "../Api/CountyData";
-import { loadControlPromise } from "../Api/AlgorithmData";
-import { loadControlDefaultParams } from "../Api/algorithmDefaultParams";
-import { serverUrl } from "../Api/server";
-import axios from "axios";
+import React, {Component} from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import { withStyles } from '@material-ui/core/styles';
+import { countyRes } from '../Api/CountyData';
+import { loadControlPromise } from '../Api/AlgorithmData';
+import { loadControlDefaultParams } from '../Api/algorithmDefaultParams';
+import { serverUrl } from '../Api/server';
+import axios from 'axios';
```

Comment:
Is this formatting from eslint?

Reply:
no it was me, does eslint prefer "?

Reply:
Yes, I've been using ESLint on script -- just don't want to put in extra work for ourselves formatting in the future, no big deal here. If you have more preferred linter for javascript, let me know! eslint has been seeming to work well for far.

Reply:
I’ll make the change before merging